include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -1402,6 +1499,53 @@ namespace internal
                                                         2 * n_q_points,
                                                       values_dofs);
           }
+        if (integration_flag & EvaluationFlags::hessians)
+          {
+            // diagonal
+            if (integration_flag & EvaluationFlags::values ||
+                integration_flag & EvaluationFlags::gradients ||
+                add_into_values_array == true)
+              eval.template hessians<0, false, true>(hessians_quad,
+                                                     values_dofs);
```

Comment:
Could you add here and below the same comments as above ("grad xx", "grad xy", ...)?

Reply:
Done!